celery_utils/__init__.py

```diff
@@ -2,6 +2,6 @@
 Code to support working with celery.
 """
 
-__version__ = '0.5.3'
+__version__ = '0.5.4'
```

Comment:
Does this actually merit a version bump?  I don't think any of the modified files are even included in the distributed package, see https://github.com/edx/edx-celeryutils/blob/master/MANIFEST.in